wordpress-mobile/WordPress-Android
# MSD - Stats Card - Navigate to `Stats -> Days`

Parent #15201 

This PR adds the implementation for the navigation from `Today's Stats` card to the `Days` tab.  

Functionalities added 
- [x] Clicking on Today's stats card navigates to stats activity (days tab)
- [x] Clicking on Today's stats card footer link button(Go to stats) navigates to stats activity (days tab)

Please find below the video of the functionality 

https://user-images.githubusercontent.com/17463767/153388129-09a0a929-9d8a-4a5d-bf96-91cd355d5658.mp4

To test:

Prerequisites
1. Launch the app
2. Select a wp com site
3. Navigate to My Site -> App Settings -> Debug Settings
4. Ensure that the MySiteDashboardPhase2FeatureConfig is to set to on
5. Restart the app if needed
6. Navigate to the My Site tab
7. Notice that Today's stats card is shown.

Test 1
1. Click on Today's stats card 
2. Stats page is shown with Day tab selected 

Test 2 
1. Click on `Go to stats` button
2. Stats page is shown with Day tab selected 

## Regression Notes
1. Potential unintended areas of impact
- `Today's stats` card is not being shown properly 
-  Existing stats redirection on clicking at `stats` button in quick links section doesn't work properly 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual Tests 

3. What automated tests I added (or what prevented me from doing so)
Unit tests 

PR submission checklist:
